v2/tools/generator/internal/codegen/pipeline/create_arm_types.go

```diff
@@ -389,3 +440,22 @@ func (c *armTypeCreator) convertObjectPropertiesForARM(t *astmodel.ObjectType, i
 
 	return result, nil
 }
+
+func skipUserAssignedIdentity(def astmodel.TypeDefinition) bool {
+	return def.Name().Name() == astmodel.UserAssignedIdentitiesTypeName
+}
+
+func requiresARMType(def astmodel.TypeDefinition) bool {
+	skipFuncs := []func(it astmodel.TypeDefinition) bool{
+		skipUserAssignedIdentity,
+	}
```

Comment:
Maybe move this to a global so it's constructed once?

Reply:
Done